build/jshint/xfail.list

```diff
@@ -62,6 +62,7 @@ apps/keyboard/test/unit/imes/jspinyin/jspinyin_test.js
 apps/keyboard/test/unit/imes/latin/worker_test.js
 apps/keyboard/test/unit/setup_engine.js
 apps/music/js/metadata.js
+apps/settings/js/jszip.js
```

Comment:
As we are planning to remove `xfail.list`, let's add the path to `.jshintignore`.